mrjonhazlett-Omaha_NE.md

```diff
@@ -13,3 +13,4 @@
 - TD Ameritrade Park (:baseball)
 - Old Market
 - Pedestrian Bridge
```

Comment:
Are there a lot of places to visit in Omaha? I think of it as a very small place.